### PR TITLE
chore: modernize code / no-var

### PR DESCRIPTION
**Pull request by Shinigami92**
***Short description of what this resolves:***

Modernize the code and use `es6` `let` and `const`

***Proposed changes:***

- Apply eslint rule [no-var](https://eslint.org/docs/rules/no-var)
- Apply eslint rule [prefer-const](https://eslint.org/docs/rules/prefer-const)

### CODE REVIEW

File: test/rules/tags-check.spec.js [hunk 1]
```diff
@@ -14,46 +14,46 @@ ruleOptions[ruldId] = {
...
-    var messages = HTMLHint.verify(code, ruleOptions)
+    const code = '<sometag></sometag>'
+    const messages = HTMLHint.verify(code, ruleOptions)
     expect(messages.length).to.be(2)
     expect(messages[0].rule.id).to.be(ruldId)
   })
   it('Should check required attributes with specifyed values', () => {
-    var code = '<sometag attrname="attrvalue" />'
-    var messages = HTMLHint.verify(code, ruleOptions)
+    let code = '<sometag attrname="attrvalue" />'
```
Comment (thedaviddias) [inline test/rules/tags-check.spec.js:55]: why let?

Comment (Shinigami92) [reply test/rules/tags-check.spec.js:55]: Because it gets reassigned 3 lines below

**Pull request merged**